ehrenamt-justiz-backend/src/test/resources/application-test.yml

```
@@ -9,10 +9,8 @@
-de:
-  muenchen:
-    ewo:
-      eai:
-        server: http://localhost:8085
-        user: test
-        password: test
+ewo:
+  eai:
+    server: http://localhost:8085
+    user: test
+    password: test
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)